lib/src/base-styles/_nx-btn.scss

```diff
@@ -90,18 +92,17 @@
   border-color: transparent;
   color: var(--nx-swatch-white);
 
-  &:focus {
-    background-color: var(--nx-swatch-red-35);
+  &:hover {
+    background-color: var(--nx-swatch-red-30);
   }
 
-  &:hover {
-    background-color: var(--nx-swatch-red-35);
+  &:focus:not(:active) {
+    outline-offset: -5px;
```

Comment:
Looks like we're using `-5px` in multiple places. We could probably use a local variable.

Reply:
> Looks like we're using `outline-offset: -5px` in multiple places. We could probably use a local variable.

Makes sense! What are your thoughts about maybe creating a @mixin as well to hold both the outline-offset & outline property to clean things up?

Reply:
> > Looks like we're using `outline-offset: -5px` in multiple places. We could probably use a local variable.
> 
> Makes sense! What are your thoughts about maybe creating a @mixin as well to hold both the outline-offset & outline property to clean things up?

My only hesitation with a mixin is the outline color. If the outline colors were to be consistent across all components, that would work well, but we have different outline colors for different components.

Reply:
> My only hesitation with a mixin is the outline color. If the outline colors were to be consistent across all components, that would work well, but we have different outline colors for different components.

Mixins can have parameters and those parameters can have default values

Reply:
> Mixins can have parameters and those parameters can have default values

Also consider `@extend` with a [placeholder selector](https://sass-lang.com/documentation/style-rules/placeholder-selectors)

Reply:
> Also consider `@extend` with a [placeholder selector](https://sass-lang.com/documentation/style-rules/placeholder-selectors)

Thanks for pointing me towards that, it seems like a better idea to use that to keep our CSS output a bit cleaner. Implemented here: [6da3385](https://github.com/sonatype/sonatype-react-shared-components/pull/983/commits/6da33851020820429f2aee544dccf6256896ebf3)

Reply:
> Thanks for pointing me towards that, it seems like a better idea to use that to keep our CSS output a bit cleaner. Implemented here: [6da3385](https://github.com/sonatype/sonatype-react-shared-components/pull/983/commits/6da33851020820429f2aee544dccf6256896ebf3)

I'd actually keep the placeholder class inside of the `_nx-btn.scss` file.  I've seen the scss compiler get weird with duplicated output when a placeholder class is imported and extended in multiple files.

Reply:
> I'd actually keep the placeholder class inside of the `_nx-btn.scss` file. I've seen the scss compiler get weird with duplicated output when a placeholder class is imported and extended in multiple files.

Sounds good! Moved here: [f100fd7](https://github.com/sonatype/sonatype-react-shared-components/pull/983/commits/f100fd732874aacc5454c65637dc62f77116be66)